CRM/Report/Form/Contact/Detail.php

```diff
@@ -608,15 +606,18 @@ public function where() {
    */
   public function clauseComponent() {
     $selectedContacts = implode(',', $this->_contactSelected);
-    $contribution = $membership = $participant = NULL;
     $eligibleResult = $rows = $tempArray = array();
     foreach ($this->_component as $val) {
       if (!empty($this->_selectComponent[$val]) &&
         ($val != 'activity_civireport' && $val != 'relationship_civireport')
       ) {
+        $tableName = str_replace('_civireport', '', $val);
```

Comment:
This is bad form to be changing the table alias through a string replacement. If you need it to be different in the SQL, change it where it is initially defined in this file.